GabrielaRamirez/EjerciciosHackerrank/src/basics/InheritanceI.java

```diff
@@ -0,0 +1,32 @@
+package basics;
+
+class Animal{
```

Comment:
According to Java standards and common practices, we should declare every class in its own source file.

https://dzone.com/articles/why-single-java-source-file-can-not-have-more-than

Reply:
ok, i just give a copy and past from the hackerrank, i'll put them on their own source